idealingua-v1/idealingua-v1-runtime-rpc-typescript/src/npmjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": false,
-  "name": "@izumi-framework/izumi-runtime-typescript",
+  "name": "undefined",
```

Comment:
@dandriushchenko Why change to undefined?

Reply:
it's now set via publish.json,
but nevermind, I change it back to be backward compatible